main.py

```diff
@@ -1,19 +1,76 @@
+from typing import Union
```

Comment:
We are using python 3.11 i think, do we still need to use Union?

How about using Union operator |

Reply:
update the code with union operator